models/ht_neuron.nestml

```diff
@@ -99,21 +99,20 @@ neuron ht_neuron_nestml:
     recordable function I_h pA = -h_g_peak * Ih_m  * ( V_m - h_E_rev )
     # The spike current is only activate immediately after a spike.
     function I_spike mV = (g_spike) ? -( V_m - E_K ) / Tau_spike : 0
-    V_m'  = ( I_Na + I_K + I_syn + I_NaP + I_KNa + I_T + I_h + I_stim ) / Tau_m + I_spike
-
+    V_m'  = ( ( I_Na * 1 pA + I_K * 1 pA + I_syn + I_NaP + I_KNa + I_T + I_h + I_stim ) / Tau_m + I_spike * 1 [pA/(ms * mV)] ) * 1 [s/nF]                    
```

Comment:
Why are some currents multiplied with units and others not? This seems to point to inconsistencies and errors elsewhere. All currents should "arrive" here with proper units, including I_spike. I would also suggest that, for easier comparison with the equation in the paper, we should have `I_spike/tau_spike` here, instead of incorporating the time constant in `I_spike`; as it is now, `I_spike` looks like a current but isn't, and that is going to cause misunderstandings later.

Note one challenge with the Hill-Tononi model: In that model, all conductances are dimensionless, which also means that currents are measured in mV. Especially, the `1 s/nF` term is not needed.